src/cascadia/TerminalSettingsModel/VsSetupConfiguration.h

```diff
@@ -167,6 +194,22 @@ namespace winrt::Microsoft::Terminal::Settings::Model
                 return GetVersion();
             }
 
+            unsigned long long GetInstallDate() const
+            {
+                return VsSetupConfiguration::GetInstallDate(inst.get());
+            }
+
+            unsigned long long GetInstallationVersion() const

```

Comment:
I'm confused what the difference between this `GetInstallationVersion` and `VsSetupConfiguration::GetInstallationVersion` is... Since they have the same name, shouldn't they also do the same thing?

Reply:
One is calling the equivalent method on the com_ptr to get and cache the value. The other is as const that simply returns the cached value. std::sort() is going to get the value multiple times so it's be inefficient to not only fetch the value but QI the setup helper.